### PR TITLE
Update mteFunctions.pas

### PR DESCRIPTION
**Pull request by Sharlikran**
- Depreciated ElementByIP
- Replaced all ElementByIP with ElementByPath
- Created Exception for ElementByIP when used
- Altered GetVersionString with Real Shelter version for easier comparison

### CODE REVIEW

**Comment by Sharlikran** [comment]
I know it says it has conflicts, but if there is a way to force it, use it as is.

**Comment by Sharlikran** [comment]
Opps I didn't notice you had made changes, I didn't rebase. Hang on

**Comment by Sharlikran** [comment]
Those seem to be the safe changes

**Comment by Sharlikran** [comment]
Are the changes here problematic?

https://github.com/TES5Edit/TES5EditScripts/commit/70c4d7719ba29fe1f95875c4d56c4e1db3f9eaba

**Comment by Sharlikran** [comment]
Also the Russian one would be intrusive because it would overwrite the English comments rather then a separate file. Unless I missed something. Also the hishutup PR might not be needed. One of those commits still uses the old xEdit hardcoded file which doesn't exist anymore.

**Comment by Sharlikran** [comment]
Although on second thought your idea was a bit better for ElementByIp, less changes for users and old scripts. I'll update that tomorrow.

File: Edit Scripts/mteFunctions.pas [hunk 1]
```diff
@@ -1264,25 +1266,16 @@ function SmallName(e: IInterface): string;
...
-  element1 := ElementByIP(e, 'Conditions\[1]\CTDA - \Function');
+  This is a function Depreciated
 }
 function ElementByIP(e: IInterface; ip: string): IInterface;
-var
-  i, index: integer;
-  path: TStringList;
 begin
-  Result := ElementByPath(e, ip);
+    raise Exception.Create('Replace ElementByIP with ElementByPath');
```
Comment (matortheeternal) [inline Edit Scripts/mteFunctions.pas]: raising an exception?  doesn't this just mean users who update mteFunctions will get an error thrown?  why would we prefer this over just calling `ElementByPath` ?

File: Edit Scripts/mteFunctions.pas [hunk 2]
```diff
@@ -173,14 +173,16 @@
...
-    v shr 16 and $FF,
-    v shr 8 and $FF
-  ]);
+    Result := Format('%d.%d.%d', [
+    //Result := Format('%sEdit version %d.%d.%d', [
+        //wbAppName,
+  	    Int(v) shr 24,
+  	    Int(v) shr 16 and $FF,
+  	    Int(v) shr 8 and $FF
+  	]);
```
Comment (matortheeternal) [inline Edit Scripts/mteFunctions.pas]: little odd that the function return is being changed here, but that's fine I guess.  I kind of would prefer if you just added a helper function to Real Shelter to do this.

**Comment by Sharlikran** [comment]
Yes I am planning on reverting the exception. It also really doesn't need to change the version check. I can test something else for that. In the end I wish the version check actually worked to see 3.1.x or higher or 4.0.x or higher as an integer rather then the presence of a string but that's something the can be worked on at a later time.

**Pull request merged**